### ISSUE TITLE
resolve version conflict in 

### ISSUE THREAD

**Issue by mkrzemien** [softwaremill/diffx#399]
An attempt to upgrade scalatest version is blocked by a version conflict:
```
[error] (docs / update) found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error] 
[error] 	* org.scala-lang.modules:scala-xml_2.13:2.1.0 (early-semver) is selected over 1.3.0
[error] 	    +- org.scalatest:scalatest-core_2.13:3.2.12           (depends on 2.1.0)
[error] 	    +- eu.timepit:refined_2.13:0.9.28                     (depends on 1.3.0)
```

See for example: https://github.com/softwaremill/diffx/pull/382

Solution proposal: exclude `org.scala-lang:scala-xml` transitive dependency on `org.scalatest` test dependencies.

**Comment by ghostbuster91** [comment]
Seems like it is no longer needed, as scala-steward was able to perform the update. We might need to do that exclusion in the future if situation repeats. Thanks for the heads up and the suggestion :+1:
